commit.go

```diff
@@ -78,15 +78,16 @@ type PushOptions struct {
 
 // Commit writes the contents of the container, along with its updated
 // configuration, to a new image in the specified location, and if we know how,
-// add any additional tags that were specified.
-func (b *Builder) Commit(ctx context.Context, dest types.ImageReference, options CommitOptions) error {
+// add any additional tags that were specified. Returns the ID of the new image
+// if successful.
+func (b *Builder) Commit(ctx context.Context, dest types.ImageReference, options CommitOptions) (string, error) {
 	policy, err := signature.DefaultPolicy(getSystemContext(options.SystemContext, options.SignaturePolicyPath))
 	if err != nil {
-		return errors.Wrapf(err, "error obtaining default signature policy")
+		return "", errors.Wrapf(err, "error obtaining default signature policy")
```

Comment:
Won't it be easier to declare `var imgID string` in the beginning of the function and return it instead of the empty string `""`?
Obviously in 126 after we get `img` we set `imgID` to `img.ID`?

WDYT?

Reply:
@ripcurld Seems like a good way of avoiding the extensive use of the `""` constant